src/components/ReportActionItem/IOUAction.js

```diff
@@ -42,7 +47,6 @@ const IOUAction = (props) => {
         <>
             <IOUQuote
                 action={props.action}
-                shouldShowViewDetailsLink={Boolean(props.action.originalMessage.IOUReportID)}
```

Comment:
BUG https://github.com/Expensify/App/pull/12762#pullrequestreview-1189779036 is happening due to the removal of this.  

Maybe rename this to `shouldAllowViewDetails`.

Reply:
``` js
const IOUQuote = props => (
    <View style={[styles.chatItemMessage]}>
        {_.map(props.action.message, (fragment, index) => (
            <Pressable
                key={`iouQuote-${props.action.reportActionID}-${index}`}
                onPress={props.shouldAllowViewDetails
                    ? props.onViewDetailsPressed
                    : () => {}}
                style={[styles.flexRow, styles.justifyContentBetween]}
            >
                <Text>
                    <Text style={[styles.chatItemMessageLink]}>
                        {/* Get first word of IOU message */}
                        {Str.htmlDecode(fragment.text.split(' ')[0])}
                    </Text>
                    <Text style={[styles.chatItemMessage]}>
                        {/* Get remainder of IOU message */}
                        {Str.htmlDecode(fragment.text.substring(fragment.text.indexOf(' ')))}
                    </Text>
                </Text>
                <Icon src={Expensicons.ArrowRight} fill={props.shouldAllowViewDetails ? themeColors.icon : themeColors.transparent} />
            </Pressable>
        ))}
    </View>
);
```

Do you have input on the best method of doing this? Three things change when a split bill is not clickable: (1) the icon is removed/transparent, (2) the cursor should no longer be a pointer, and (3) clicking should not open the modal. 
This solution works for (1) and (3) but the cursor is still in focus mode because of the pressable parent component. Should I use a function here instead of 3 ternaries?

Reply:
(1) the icon is removed/transparent, :heavy_check_mark: 
 (2) the cursor should no longer be a pointer :heavy_check_mark: 
 (3) clicking should not open the modal. :heavy_check_mark: 
 
 All should be done. 
 
 For 2, 
 1. We can't use disabled prop, it disables selection. 
 2. Instead, we can use `focusable={false}` and apply a new style `{cursor: 'default'}`.